src/main/java/uk/nhs/ctp/config/interceptors/AuditRestClient.java

```diff
@@ -2,13 +2,13 @@
 
```

Comment:
If AuditFhirClientInterceptor has moved to /audit, should this not also be moved?

Reply:
Not this one as it's not a common requirement (only the EMS needs it), and `/audit` only contains things that will be moved to `common`. Additionally, the story only required the capture of FHIR requests logs.